README.md

```diff
@@ -144,6 +145,38 @@ $ spanner-cli -p myproject -i myinstance -d mydb -e 'SELECT * FROM users;' -t
 +----+------+--------+
 ```
 
+### Directed reads mode
+
+spanner-cli now supports directed reads, a feature that allows you to read data from a specific replica of a Spanner database. 
+To use directed reads with spanner-cli, you need to specify the `--directed-read` flag.
+The `--directed-read` flag takes a single argument, which is the name of the replica that you want to read from.
+The replica name can be specified in one of the following formats:
+
+- `<replica_location>`
+- `<replica_location>:<replica_type>`
+
+The `<replica_location>` part of the replica name specifies the region where the replica is located such as `us-central1`, `asia-northeast2`.  
+The `<replica_type>` part of the replica name specifies the type of the replica either `READ_WRITE` or `READ_ONLY`. 
+ 
+```
+$ spanner-cli -p myproject -i myinstance -d mydb --directed-read us-central1
+
+$ spanner-cli -p myproject -i myinstance -d mydb --directed-read us-central1:READ_ONLY
+
+$ spanner-cli -p myproject -i myinstance -d mydb --directed-read asia-northeast2:READ_WRITE
+```
+
+Directed reads are only effective for single queries or queries within a read-only transaction.
+Please note that directed read options do not apply to queries within a read-write transaction.
+
+> [!NOTE]
+> If you specify an incorrect region or type for directed reads, directed reads will not be enabled and [your requsts won't be routed as expected](https://cloud.google.com/spanner/docs/directed-reads#parameters). For example, in a multi-region configuration `nam3`, if you mistype `us-east1` as `us-east-1`, the connection will succeed, but directed reads will not be enabled. 
+>
+> To perform directed reads to `asia-northeast2` in a multi-region configuration `asia1`, you need to specify `asia-northeast2:READ_WRITE`.
```

Comment:
> you need to specify `asia-northeast2:READ_WRITE`.

you need to specify `asia-northeast2` or `asia-northeast2:READ_WRITE`?

Reply:
fix